public/javascripts/components/conversation/conversation.js

```diff
@@ -84,18 +75,19 @@ window.ChubboChat.components.conversation = Vue.extend({
           sender: 'bot'
         });
       } else if (me.surveyQuestions.length === 0) {
+        if (!me.isSurveyComplete) {
+          //say bye
+          me.messages.push({
+            text: 'Thanks for taking the survey! Visit chubbo-chat.herokuapp.com/#!/dashboard to create your own survey!',
```

Comment:
Can we put this text into some kind of const? I think it will make it easier to track down, change, and reuse later